### PR TITLE
fix(ci): exclude testE2E/ from coverage analysis

### PR DESCRIPTION
**Pull request by justinmk3**
- fix(ci): exclude `testE2E/**` and `testLint/**` from coverage analysis
- rename `integrationTest/` => `testInteg/`

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .nycrc.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
     "all": true,
     "include": ["src/**", "dist/src/**"],
-    "exclude": ["test/**", "integrationTest/**", "testFixtures/**"]
+    "exclude": ["test/**", "testE2E/**", "testLint/**", "testInteg/**", "testFixtures/**"]
```
Comment (justinmk3) [inline .nycrc.json:4]: should these be prefixed with `src/` ?
```suggestion
    "exclude": ["src/test/**", "src/testE2E/**", "src/testLint/**", "src/testInteg/**", "src/testFixtures/**"]
```

**Review by nkomonen-amazon — APPROVED** [review]

File: .nycrc.json (hunk 1 above)
Comment (JadenSimon) [reply .nycrc.json:4]: I think the patterns are applied relative to `include` but not 100% sure

Comment (justinmk3) [reply .nycrc.json:4]: that seems to be what the examples in https://github.com/istanbuljs/nyc#selecting-files-for-coverage imply, but it doesn't clearly say.

**Pull request merged**